pkg/chargeback/http.go

```diff
@@ -440,19 +440,26 @@ type ReportResultEntry struct {
 }
 
 type ReportResultValues struct {
-	Name  string      `json:"name"`
-	Value interface{} `json:"value"`
+	Name        string      `json:"name"`
+	Value       interface{} `json:"value"`
+	TableHidden bool        `json:"tableHidden"`
+	Unit        string      `json:"unit"`
```

Comment:
Lets set `json:"unit,omitempty"` so the `unit` field isn't in the output unless `unit` is non-empty.